### ISSUE TITLE
Header should not have H1 component

### ISSUE THREAD

**Issue by jacquicosa** [collegescore/frontend#17]
For the Header, make the College Score logo text be an svg with alt text rather than text text. The H1 on the home page should be the hero, not the header.
